contracts/pool/src/methods/liquidate.rs

```diff
@@ -105,8 +107,8 @@ fn do_liquidate(
         let s_token = STokenClient::new(env, &reserve.s_token_address);
         let debt_token = DebtTokenClient::new(env, &reserve.debt_token_address);
```

Comment:
Looks like we can move this instantiation under else branch below

Reply:
Good idea! I moved the `DebtTokenClient`, but kept the `STokenClient` since it's used in both `if` and `else`` statements.